sh/tests/lock_file.sh

```
@@ -0,0 +1,45 @@
+        break
+    fi
+    echo "Waiting for gitbak process to terminate ($i/5)..."
+    sleep 1
+done
+
+if ps -p $GITBAK_PID1 > /dev/null 2>&1; then
+    echo "Process didn't terminate gracefully, force killing..."
+    kill -9 $GITBAK_PID1 2>/dev/null
+fi
+
+cd - >/dev/null
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)